packages/wonder-stuff-core/src/values.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+/**
+ * Return an array of the enumerable properties of an object.
+ *
+ * @param {$ReadOnly<{[mixed]: V}>} obj The object for which the values are
+ * to be returned.
+ * @returns {Array<V>} An array of the enumerable properties of an object.
```

Comment:
Nit: this should probably say "enumerable properties of _the_ object", or perhaps just "enumerable property values" (since the values, not the properties, are returned).

Reply:
I based this language off the language that is documenting Object.values